### PR TITLE
add parameters 24, 30

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: docs/parameter-types.md [hunk 1]
```diff
@@ -318,6 +318,37 @@ It is a `4`-byte unsigned int BE.
 
 Message hex dump with LRC: `03 0a 17 00 00 00 cc 83 00 00 07 e7 e4`
 
+## Enable absolute data
```
Comment (DarkPark) [inline docs/parameter-types.md:321]:
```suggestion

## Enable absolute data
```

File: docs/parameter-types.md [hunk 2]
```diff
@@ -404,3 +435,43 @@ It is a `4`-byte unsigned int BE.
...
+### Examples
+
+#### enable absolute data for 2 channel:
+
+| Field               | Value | Hex          |
+| ------------------- | ----- | ------------ |
+| command id          | `3`   | `0x03`       |
+| command size        | `3`   | `0x02`       |
+| parameter type      | `30`  | `0x1e`       |
+| absolute data state | `1`   | `0x01`       |
```
Comment (DarkPark) [inline docs/parameter-types.md:474]: wrong field

File: docs/parameter-types.md [hunk 3]
```diff
@@ -404,3 +435,43 @@ It is a `4`-byte unsigned int BE.
...
+| `1`   | device send absolute value from channel        |
+
+### Examples
+
+#### enable absolute data for 2 channel:
+
+| Field               | Value | Hex          |
+| ------------------- | ----- | ------------ |
+| command id          | `3`   | `0x03`       |
+| command size        | `3`   | `0x02`       |
```
Comment (DarkPark) [inline docs/parameter-types.md:472]:
```suggestion
| command size        | `3`   | `0x03`       |
```

File: docs/parameter-types.md (hunk 1 above)
Comment (DeadByElpy) [reply docs/parameter-types.md:322]: fixed

File: docs/parameter-types.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**